webpack.base.config.js

```diff
@@ -40,34 +40,24 @@ const baseConfig = {
       },
       {
         test: /\.svg$/,
-        loader: 'svg-inline',
+        loader: 'svg-inline-loader',
       },
       {
         test: /\.ejs/,
         loader: 'underscore-template-loader',
       },
     ],
   },
-  postcss() {
-    return [
-      autoprefixer,
-    ];
-  },
   devtool: 'source-map',
   plugins: [
     new webpack.IgnorePlugin(/^\.\/locale$/, /moment$/),
-    new webpack.optimize.DedupePlugin(),
-    new webpack.optimize.OccurenceOrderPlugin(),
     new webpack.EnvironmentPlugin(Object.keys(process.env)),
-    new ExtractTextPlugin(
-      'assets-honestly/styles-[contenthash:base64:5].css',
-      { allChunks: true }
-    ),
+    new webpack.LoaderOptionsPlugin({ options: { context: __dirname, postcss: [autoprefixer] } }),
+    new ExtractTextPlugin({ allChunks: true, filename: 'assets-honestly/styles-[contenthash:base64:5].css', ignoreOrder: true }),
```

Comment:
This same `ExtractTextPlugin` change needs to go in `webpack.dev.browser.config.js`